### DISCUSSION TITLE
Preview d'un fichier attaché

### DISCUSSION THREAD

**Discussion by maximpouliot** [MTESSDev/FRW#113] [Annonces]
Lors de notre lab client, un commentaire a aussi été de donner la possibilité à l'utilisateur de visualiser le fichier qu'il a attaché au formulaire afin de s'assurer qu'il nous envoi la bonne documentation avant de confirmer. 

Est-ce possible? 

Merci!

**Comment by florentarneodo** [comment]
Réponse de Yanick : c'est un besoin dans le backlog du MESS
(pour les images, c'est possible de les éditer, rogner, etc. dans le ctl de pièce jointe)
